.github/CONTRIBUTING.md

```diff
@@ -81,6 +99,16 @@ $ make html
 
 The built documentation can then be found in `docs/_build/html/`.
 
+To file a pull request, create a new branch on top of the upstream repository:
+
+```console
+git checkout -b my_topical_branch upstream/main
```

Comment:
The `console` lexer is designed to highlight commands with leading prompt (keeping the optional output uncolored).

```suggestion
$ git checkout -b my_topical_branch upstream/main
```